dashboard/lib/api/v1/pd/workshop_score_summarizer.rb

```diff
@@ -285,6 +287,25 @@ def generate_response_averages(responses, response_sums, facilitator_specific_re
     response_summary
   end
 
+  def coerce_old_surveys_to_new_format(responses, workshops)
```

Comment:
Add a function header briefly explaining the old vs. new format and what this does